spec/views/idv/doc_auth/upload.html.erb_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe 'idv/doc_auth/upload.html.erb' do
+  let(:flow_session) { {} }
+  let(:user_fully_authenticated) { true }
+
+  before do
+    allow(view).to receive(:flow_session).and_return(flow_session)
+    allow(view).to receive(:user_fully_authenticated?).and_return(user_fully_authenticated)
+    allow(view).to receive(:url_for).and_return('https://www.example.com/')
+    allow(view).to receive(:user_signing_up?).and_return(false)
+  end
+
+  context 'with liveness enabled' do
+    it 'renders liveness content' do
+      allow(view).to receive(:liveness_checking_enabled?).and_return(true)
+      render template: 'idv/doc_auth/upload.html.erb'
+
+      expect(rendered).to include(CGI.escapeHTML(t('doc_auth.headings.upload_liveness_enabled')))
```

Comment:
Maybe instead of CGI escaping we could strip tags to just get the plain text?

Example: https://github.com/18F/identity-idp/blob/master/spec/views/sign_up/completions/show.html.erb_spec.rb#L26-L31

Reply:
I added the escaping because the strings `We'll collect information about you by...` have an apostrophe that becomes `We&#39;ll collect information about you by...` in the HTML. `strip_tags` doesn't work for this case unfortunately, is there a better way to handle this other than escaping?

I was also a bit overzealous in my copy pasting, since only the `doc_auth.info.upload` strings need to be escaped, so I've removed the escaping from the other lines.

Reply:
Oh I see now --- I *think* the `have_content` matcher might work and do the escaping/translating for us: https://github.com/18F/identity-idp/blob/master/spec/views/users/delete/show.html.erb_spec.rb#L22

But otherwise, this is probably fine as-is for a one-off

Reply:
Oh yep, it sure does, switched to that